src/test-gui/java/de/qabel/desktop/ui/about/AboutPage.java

```diff
@@ -0,0 +1,33 @@
+package de.qabel.desktop.ui.about;
+
+import com.sun.javafx.robot.FXRobot;
+import de.qabel.desktop.ui.AbstractPage;
+import org.testfx.api.FxRobot;
+
+import static org.junit.Assert.assertTrue;
+
+public class AboutPage extends AbstractPage {
+    private AboutController controller;
+
+    public AboutPage(FXRobot baseFXRobot, FxRobot robot, AboutController controller) {
+        super(baseFXRobot, robot);
+        this.controller = controller;
+    }
+
+    public void btnShowPopup() {
```

Comment:
no abbreviations please. It's just a button. But methods on a page should better be named after actions (like ::showPopup(), ::close(), etc.)